sql/step_04_all_following_steps.sql

```diff
@@ -92,4 +92,5 @@ source step_95_missing_user_related_fk.sql;
 source step_96_user_settings_table.sql;
 source step_97_fix_default_values.sql;
 source step_98_reset_permissions.sql;
-source step_99_telemetry.sql;
\ No newline at end of file
+source step_99_telemetry.sql;
+source step_100_add_delete_btn_in_medicalstock.sql;
```

Comment:
Please call it like that, we will rename it just before the merge. Thanks!

```suggestion
source step_aXXX_add_delete_btn_in_medicalstock.sql;
```